### PR TITLE
Unit Test Send Mail

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Unit test send mail and fix debug-bar
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: TI-3.3

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Backend/Borrowings/AdminSendMailsTest.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    }
+
+    /**
+     * A Dusk test route to page list Borrowings.
+     *
+     * @return void
+     */
+    public function testRouteListBorrowings()
+    {
+        $user = $this->makeAdminToLogin();
```
Comment (at-anhngo) [inline tests/Browser/Pages/Backend/Borrowings/AdminSendMailsTest.php]: nên để trong constructor của class để đỡ phải create data nhiều lần

**Review by ghost — COMMENTED** [review]
check my comment

File: tests/Browser/Pages/Backend/Borrowings/AdminSendMailsTest.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            $browser->loginAs($user)
+                    ->visit('/admin/borrowings')
+                    ->resize(1600, 2000)
+                    ->click('#1')
+                    ->assertSee($borrower->users->name)
+                    ->pause(1000)
+                    ->assertSee('Confirm Send Mail')
+                    ->press('OK')
+                    ->pause(3000)                    
+                    ->assertSee('Send Mail Success');
```
Comment (at-anhngo) [inline tests/Browser/Pages/Backend/Borrowings/AdminSendMailsTest.php]: thêm test trong db có thời gian gửi mail nữa

**Comment by at-anhngo** [comment]
LGTM!!!

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**